cob_bringup/robots/cob4-3.xml

```diff
@@ -44,9 +44,6 @@
 			<arg name="robot" value="$(arg robot)" />
 		</include>
 		<include file="$(find cob_script_server)/launch/script_server.launch" />
-		<!--include file="$(find cob_bringup)/tools/base_collision_observer.launch" >
```

Comment:
@ipa-nhg: why did you remove the base_collision_observer? IMHO this does something different than the collision_monitor: providing the safe twist command topic for the base